articles/spring-cloud/spring-cloud-faq.md

```diff
@@ -34,6 +34,9 @@ Security and privacy are among the top priorities for Azure and Azure Spring Clo
 
 East US, East US 2, Central US, South Central US, West US 2, West Europe, North Europe, UK South, Southeast Asia and Australia East.
 
+### Is any customer data stored outside of the specified region?
+
+Azure Spring Cloud is a regional service. All customer data in Azure Spring Cloud is stored to multiple regions within the same geo of the specified region for redundancy. Learn more about geo and region from [Data residency in Azure](https://azure.microsoft.com/en-us/global-infrastructure/data-residency/).
```

Comment:
@xscript - Hello Kevin, overall a small and a quick merge. At first I questioned the usage of the word geo but it seems in line with terminology from the data residency content.

Reply:
#sign-off